saos-webapp/src/main/webapp/WEB-INF/view/judgment/judgmentDetails.jsp

```diff
@@ -205,7 +207,10 @@ $(document).ready(function() {
 										</c:forEach>
 									</c:when>
 									<c:otherwise>
-										<a href="${contextPath}/search?signature=${refCourtCase.caseNumber}">
+										<c:url var="refCourtCaseUlrSearch" value="/search" >
```

Comment:
Do you know that in case of a user without cookies enabled, c:url will add a sessionId to the url to enable url rewriting?